src/main/kotlin/org/rust/lang/utils/RsDiagnostic.kt

```diff
@@ -1455,3 +1473,21 @@ private fun getConflictingNames(element: PsiElement, vararg tys: Ty): Set<RsQual
         emptySet()
     }
 }
+
+private fun createReplaceFeatureFix(element: RsElement, newFixName: String, oldFixName: String): LocalQuickFix? {
+    val project = element.project
+    val crateRoot = element.crateRoot
+    val oldAttr = RsFeatureIndex.getFeatureAttributes(project, oldFixName)
+        .find { it.crateRoot == crateRoot }
+        ?: return null
```

Comment:
It won't work when `min_const_generics` will be stabilized because there won't be `#![feature(min_const_generics)]` anywhere

Reply:
I suppose in such cases we should provide `AddFeatureAttributeFix` fix instead